.circleci/config/jobs/@jobs.yml

```diff
@@ -325,7 +325,7 @@ deploy_dev:
           LATEST_TAG_WITH_V=`git describe --abbrev=0 --tags ${CIRCLE_SHA1}` &&
           LATEST_TAG=${LATEST_TAG_WITH_V:1}-approved &&
           ./scripts/slack/notify_deploy.py $LATEST_TAG dev &&
-          curl -i -H "Accept: application/json" -H "Content-Type: application/json" \
+          curl -s --fail --show-error -i -H "Accept: application/json" -H "Content-Type: application/json" \
```

Comment:
cool, so I see `show-error` will send the error to `stderr` and that's how it will fail in circle? :-)

Reply:
no, the show-error is only to print the error to stdout :) the `--fail` is the one that makes curl return with a failure if the http status code is not 200